test/functional/lua/uv_spec.lua

```diff
@@ -0,0 +1,53 @@
+-- Test suite for testing interactions with API bindings
+local helpers = require('test.functional.helpers')(after_each)
+local funcs = helpers.funcs
+local meths = helpers.meths
+local clear = helpers.clear
+local sleep = helpers.sleep
+local eq = helpers.eq
+
+before_each(clear)
+
+describe('vim.uv', function()
+
+  it('version', function()
+    eq(72961, funcs.luaeval('vim.uv.version()'))
+    eq("1.29.1", funcs.luaeval('vim.uv.version_string()'))
```

Comment:
hard coding the version here is probably not so useful. Maybe could check it matches a regex (or lua pattern) for semver-like version.

Reply:
This just to check nvim use right luv/uv version, this should be changed after luv update every time.  Widening inspection conditions is good choice?